source/Halibut.Tests/SerializerFixture.cs

```diff
@@ -1,25 +1,24 @@
-
-using System;
 using System.Threading.Tasks;
 using FluentAssertions;
+using Halibut.Logging;
 using Halibut.Tests.Support;
 using Halibut.TestUtils.Contracts;
 using NUnit.Framework;
 
 namespace Halibut.Tests
 {
-    public class WebSocketUsageFixture : BaseTest
```

Comment:
Git is making the changes hard to read as it's a move :( 

This fixture contains the 2 serializers is kept up to date tests we don't really understand the reason for having!!